priv/xcodegen/Sources/TemplateApp/ReconnectingView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ReconnectingView.swift
+//  ReconnectingView
+//
+
+import SwiftUI
+
+struct ReconnectingView: View {
+    var content: AnyView
+    var isReconnecting: Bool
+    
+    var body: some View {
+        content
+            .safeAreaInset(edge: .top) {
+                if isReconnecting {
+                    VStack {
+                        Label("No Connection", systemImage: "wifi.slash")
+                            .bold()
+                        Text("Reconnecting")
+                            .foregroundStyle(.secondary)
+                    }
+                        .font(.caption)
+                        .padding(8)
+                        .frame(maxWidth: .infinity)
+                        #if os(watchOS)
+                        .background(.background)
+                        #else
+                        .background(.regularMaterial)
+                        #endif
+                        .transition(.move(edge: .top).combined(with: .opacity))
+                }
+            }
+            .animation(.default, value: isReconnecting)
+    }
+}
+
+//#Preview {
+//    ReconnectingView(
+//        content: Text("Test"),
```

Comment:
@carson-katri for this I was getting an error about how `Text` cannot be converted to `AnyView`. I thought that `AnyView` would match on any view struct?

<img width="725" alt="Screenshot 2024-02-02 at 9 09 15 AM" src="https://github.com/liveview-native/liveview-client-swiftui/assets/18524/c5b4387c-8fad-4b1b-8ce1-b612dfb21447">